src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -179,6 +179,19 @@ protected override int GetIndexOfReferencedAssembly(AssemblyIdentity identity)
             return -1;
         }
 
+        protected override int GetIndexOfReferencedAssembly(AssemblyIdentity identity, AssemblyIdentityComparer comparer)
```

Comment:
Can we implement the one without a comparer in terms of the one with the comparer?